types/node/fs/promises.d.ts

```diff
@@ -30,6 +30,10 @@ declare module 'fs/promises' {
         Mode,
         WatchOptions,
     } from 'node:fs';
+    interface FileChangeInfo {
+        eventType: string;
```

Comment:
Can you use the 2 possible event types defined at 

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/10bd2a25228916286db2612946e9001740a24bd3/types/node/fs.d.ts#L2794

Extract them into a type, import and re-use it here?

Reply:
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/55033/commits/646a81331f881e98509899c000bb959a35c69f45